ts/index.ts

```diff
@@ -1673,14 +1694,8 @@ export default class Gossipsub extends EventEmitter {
         })
 
         // floodsub peers
-        peersInTopic.forEach((id) => {
-          const peerStreams = this.peers.get(id)
-
-          if (
-            peerStreams &&
-            peerStreams.protocol === constants.FloodsubID &&
-            this.score.score(id) >= this.opts.scoreThresholds.publishThreshold
-          ) {
```

Comment:
Some notes
+ If there is no floodsub peers, we save a loop through topic peers
+ One difference is the check of `peersInTopic.has(id)`, master has this but rust-libp2p does not have